crates/bevy_ui/src/ui_node.rs

```diff
@@ -111,7 +111,7 @@ impl ComputedNode {
     /// The order of the node in the UI layout.
     /// Nodes with a higher stack index are drawn on top of and receive interactions before nodes with lower stack indices.
     ///
-    /// Automatically calculated by [`super::layout::ui_layout_system`].
+    /// Automatically calculated in [`super::UiSystems::Stack`].
```

Comment:
This isn't changed here, but I think we shouldn't be showing links including `crate` and `super` in user-facing documentation.

Reply:
```suggestion
    /// Automatically calculated in [`UiSystems::Stack`](super::UiSystems::Stack).
```